pylot/perception/detection/efficientdet_operator.py

```diff
@@ -75,17 +75,17 @@ def load_serving_model(self, model_name, model_path, gpu_memory_fraction):
         detection_graph = tf.Graph()
         with detection_graph.as_default():
             # Load a frozen graph.
-            graph_def = tf.GraphDef()
-            with tf.gfile.GFile(model_path, 'rb') as f:
+            graph_def = tf.compat.v1.GraphDef()
+            with tf.io.gfile.GFile(model_path, 'rb') as f:
                 graph_def.ParseFromString(f.read())
                 tf.import_graph_def(graph_def, name='')
-        gpu_options = tf.GPUOptions(
+        gpu_options = tf.compat.v1.GPUOptions(
             allow_growth=True,
             visible_device_list=str(self._flags.obstacle_detection_gpu_index),
             per_process_gpu_memory_fraction=gpu_memory_fraction)
-        return model_name, tf.Session(
+        return model_name, tf.compat.v1.Session(
```

Comment:
Can you check that the new EfficientDet checkpoints work and if so, delete this operator?